utils/build/docker/golang/install_ddtrace.sh

```diff
@@ -14,7 +14,7 @@ elif [ -e "/binaries/golang-load-from-go-get" ]; then
 
 else
     echo "Installing production dd-trace-version"
-    go get -v -d -u gopkg.in/DataDog/dd-trace-go.v1
+    go get -v -d gopkg.in/DataDog/dd-trace-go.v1
```

Comment:
```suggestion
    go get -v gopkg.in/DataDog/dd-trace-go.v1@latest
```

-d flag is deprecated and latest is necessary, otherwise it will use the version from the `go.mod` (which is `v1.69.1` right now, not the latest published one).